lib/l10n/intl_fr.arb

```diff
@@ -0,0 +1,27 @@
+{
+  "@@locale": "fr",
+  "backButtonTooltip": "Retour",
+  "closeButtonLabel": "Fermer",
+  "searchLabel": "Rechercher",
+  "closeNavigationTooltip": "Fermer le navigateur",
+  "openNavigationTooltip": "Ouvrir le navigateur",
+  "clickToSearch": "Cliquez pour rechercher",
+  "modalBarrierDismissLabel": "Annuler",
+  "minimizeWindowTooltip": "Réduire",
+  "restoreWindowTooltip": "Restaurer",
+  "closeWindowTooltip": "Fermer",
+  "dialogLabel": "Dialogue",
+  "cutActionLabel": "Couper",
+  "copyActionLabel": "Copier",
+  "pasteActionLabel": "Coller",
+  "selectAllActionLabel": "Tout sélectionner",
+  "newTabLabel": "Ajouter un nouvel onglet",
+  "closeTabLabelSuffix": "Fermer l'onglet",
+  "scrollTabBackwardLabel": "Défiler vers l'arrière",
+  "scrollTabForwardLabel": "Défiler vers l'avant",
+  "noResultsFoundLabel": "Aucun résultat trouvé",
+  "copyActionTooltip": "Copier le contenu sélectionné dans le presse-papiers",
+  "cutActionTooltip": "Couper le contenu sélectionné et le mettre dans le presse-papiers",
+  "pasteActionTooltip": "Coller le contenu du presse-papiers à la position actuelle",
```

Comment:
```suggestion
  "copyActionTooltip": "Copier le contenu sélectionné dans le presse-papier",
  "cutActionTooltip": "Couper le contenu sélectionné et le mettre dans le presse-papier",
  "pasteActionTooltip": "Coller le contenu du presse-papier à la position actuelle",
```

Sorry, I didn't see it the first time

Reply:
Btw, I checked and both versions are syntactically correct (thank you French...) So leave it like that.

Reply:
Btw 2, I see on wikipedia that `presse-papiers` with the **s** refers to the old spelling. So you can still apply the change

Reply:
Well, if there is a newer spelling we can adopt it! Thanks @WinXaito